dist/okapi.sh

```diff
@@ -46,7 +53,7 @@ parse_okapi_conf()  {
          chmod 600 "${PID_DIR}/okapi-runtime.conf"
          # include postgres connection arguments only if defined
          jq -n --arg WARNING "AUTOMATICALLY CREATED FILE, DO NOT EDIT!"    \
-               --argjson dockerRegistries  ${docker_registries:-[\{\}]}    \
+               --argjson dockerRegistries  "${docker_registries:-[\{\}]}"  \
```

Comment:
@julianladisch in my testing using Ubuntu Focal, if `$docker_registries` is undefined, this fails if quoted (though it works if `$docker_registries` is defined). I confess I'm not sure why. Did it work in your testing?

Reply:
Good catch!
This is correct: `"${docker_registries:-[{\}]}"`
When using quotes only the closing } needs to be masked.

```
$ docker run --rm=true -it ubuntu:focal
root@e4921c64df45:/# apt-get update; apt-get install -y jq
root@e4921c64df45:/# jq -n --argjson dockerRegistries "${docker_registries:-[{\}]}" "{ \$dockerRegistries }"                                   {
  "dockerRegistries": [
    {}
  ]
}
```